src/stack.js

```diff
@@ -1,5 +1,31 @@
 'use strict'
 
 export default class Stack {
-  // your code here
+  constructor(){
+    this._storage = []
```

Comment:
Naming: I'd call this something like `elements` instead of `storage` because all variables are essentially storage.